worker/runtime/container.go

```diff
@@ -325,7 +335,9 @@ func procID(gdnProcSpec garden.ProcessSpec) string {
 	return id
 }
 
-func setupContainerdProcSpec(gdnProcSpec garden.ProcessSpec, procSpec *specs.Process) {
+func (c *Container) setupContainerdProcSpec(gdnProcSpec garden.ProcessSpec, containerSpec specs.Spec) (specs.Process, error) {
```

Comment:
Since this method doesn't rely on the `*Container`, do we want to want add behaviour on `Container` as opposed to keeping this a `static` method ?

Reply:
The method now uses Container's RootFSManager [here](https://github.com/concourse/concourse/blob/ac548408ddf8f7f0777eed85b78bca75f2875aab/worker/runtime/container.go#L366). So we thought it would be best to make it a method of `*Container`.

Do you feel it would be better to pass the `c.RootFSManager` in the signature?